elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1338,42 +1299,13 @@ protected void addInverseRelation(String relationName, Object relationValue) {
     }
 
     private <A extends Annotation> void checkFieldAwarePermissions(Class<A> annotationClass) {
-        requestScope.checkFieldAwarePermissions(annotationClass, this);
+        // TODO: Need ChangeSet support
+        requestScope.getPermissionManager().checkFieldAwarePermissions(this, null, annotationClass);
     }
 
     private <A extends Annotation> void checkFieldAwarePermissions(Class<A> annotationClass, String fieldName) {
-        requestScope.checkFieldAwarePermissions(annotationClass, this, fieldName);
```

Comment:
We could, but as a personal opinion I think this makes the rest of the `PersistentResource` code easier to read. Rather than passing the hardcoded values in each time (i.e. `this` and `null`), we can let this function do the work instead.